tests/functional/labels.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'require'
+], function (intern, registerSuite, assert, require) {
+  'use strict';
+
+  var url = function (num) {
+    return intern.config.siteRoot + '/issues/' + num;
+  };
+
+  registerSuite({
+    name: 'labels',
+
+    'label gear is visible': function () {
+      return this.remote
+        .setFindTimeout(intern.config.wc.pageLoadTimeout)
+        .get(require.toUrl(url(100)))
+        .sleep(10000)
+        .findByCssSelector('.LabelEditor-launcher span.wc-sronly')
+        .getVisibleText()
```

Comment:
I think this is going to fail because you're trying to get the visible text for an element that is positioned in outer space for screen readers. Why not find the element that is the wrapper for the label editor (`.Label-wrapper`) check [`isDisplayed()`](https://theintern.github.io/leadfoot/Element.html#isDisplayed)? We could then log out and make sure it's not visible, etc.